mlflow/server/js/src/utils/Utils.js

```diff
@@ -150,7 +150,7 @@ class Utils {
         url = url + "/src/master/" + bitbucketMatch[3];
       }
     } else if (gerritMatch) {
-        url = "https://review.crto.in/gitweb?p=" + gerritMatch[2];
+      url = "https://review.crto.in/gitweb?p=" + gerritMatch[2];
```

Comment:
You should remove this change from your PR... willl be painful to merge upstream

Reply:
but this are lines of code that we introduced right? .. stuff related to handling gerrit urls.

Reply:
Yes. Just if you want to cherry pick it to a branch to master, you will have to remove it. Not a big issue.

Reply:
yes.. but you are right .. let's not complicate things.